lib/routes/academiaedu/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```

Comment:
Please use namespace `academia` instead of `academiaedu`.
docs: https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace